internal/matchers/matchers.go

```diff
@@ -20,8 +20,11 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"log"
 	"reflect"
 
+	"github.com/davecgh/go-spew/spew"
+
```

Comment:
nit: remove empty line + goimports

Reply:
I'm not sure why but my Goimports on Goland always leaves PRs in this state - I need to get to the bottom of it.

Reply:
The problem I had a while back was that when imports very added automatically when coding (e.g. just using the package) it added them like this. 

Since a while / a few versions that problem is gone (on my machine :))

Reply:
The problem with goimports is that it just doesn't enforce order across blocks. It will split blocks but it will never merge them.

Reply:
I've been tracking this issue for a while: https://github.com/golang/go/issues/20818